test/remote/check_tests.js

```diff
@@ -28,9 +28,15 @@ test(
 
 var client = restify.createJsonClient({
   url: 'http://127.0.0.1:' + config.listen.port
-});
+})
+
+var ACTIONS = [
+  'accountCreate',
+  'accountLogin',
+  'passwordChange'
+]
 
-['accountCreate', 'accountLogin', 'passwordChange'].forEach(function (action) {
+ACTIONS.forEach(function (action) {
```

Comment:
![Happy to help!](http://33.media.tumblr.com/5cdc86d7ac4f2f76e2179f4828500647/tumblr_inline_nkx7n09n531s0m7nr.gif)